src/main/java/step3/racingcarWinner/domain/RacingGame.java

```diff
@@ -0,0 +1,88 @@
+package step3.racingcarWinner.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingGame {
+
+    private static final int DEFAULT_CAR_LOCATION = 0;
+    private static final int RANDOM_BOUND = 10;
+    private static final int MOVE_MIN_VALUE = 4;
+    private static final String SPLIT_REGX = ",";
+    private int maxPosition;
+    private List<Car> carList;
+
+    public RacingGame(String inputName) {
+        readyOnCarTrack(inputName);
+    }
+
+    private void readyOnCarTrack(String inputName) {
+        String[] carNameArr = parseCarNameArr(inputName);
+        this.carList = createCarList(carNameArr);
+    }
+
+    public List<Car> run() {
+        for (Car car : this.carList) {
+            moveCar(car);
+        }
+        return this.carList;
+    }
+
+    public String[] parseCarNameArr(String inputName) {
+        return inputName.split(SPLIT_REGX);
+    }
+
+    public boolean isMove(int arg) {
```

Comment:
`RacingGame` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private boolean isMove(int arg) {
```